### PR TITLE
Navbar for mobile

### PR DESCRIPTION
**Pull request by YaronMiro**
#58 
@amitaibu  Ready for review
Example:
nav2

### CODE REVIEW

File: src/elm/App.elm [hunk 1]
```diff
@@ -499,22 +499,43 @@ navbarLoggedIn address model =
...
-                ]
-              ]
-          ]
-      ]
+    node "nav"
+        [ class "navbar navbar-default" ]
+        [ div
+            [ class "container-fluid" ]
+            [ div [ class "navbar-header" ] [ navCollapseButton ]
+              , div
```
Comment (amitaibu) [inline src/elm/App.elm]: wrong indentation

File: src/elm/App.elm [hunk 2]
```diff
@@ -499,22 +499,43 @@ navbarLoggedIn address model =
...
+                  [ class "collapse navbar-collapse", id "main-nav"]
```
Comment (amitaibu) [inline src/elm/App.elm]: multiple lines

File: src/elm/App.elm [hunk 3]
```diff
@@ -499,22 +499,43 @@ navbarLoggedIn address model =
...
+                  [ ul
+                      [ class "nav navbar-nav"]
+                      [ li [] [ a [ hrefVoid, onClick address (SetActivePage User) ] [ text "My account"] ]
+                        , li [] [ a [ hrefVoid, onClick address (SetActivePage <| Event Nothing) ] [ text "Events"] ]
```
Comment (amitaibu) [inline src/elm/App.elm]: wrong indentation

File: src/elm/App.elm [hunk 4]
```diff
@@ -499,22 +499,43 @@ navbarLoggedIn address model =
...
+                        , li [] [ a [ hrefVoid, onClick address (SetActivePage Article) ] [ text "Articles"] ]
+                        , li [] [ a [ href "#!/error-page"] [ text "PageNotFound (404)"] ]
+                        , li [] [ a [ hrefVoid, onClick address Logout ] [ text "Logout"] ]
+                      ]
+                  ]
+            ]
+        ]
+
 
 myStyle : List (String, String)
```
Comment (amitaibu) [inline src/elm/App.elm]: can we kill this?

File: src/elm/App.elm [hunk 5]
```diff
@@ -499,22 +499,45 @@ navbarLoggedIn address model =
...
+
+
+    navCollapseButton =
+      let
+        iconBar index =
+          span [ class "icon-bar" ] []
+
+      in
+        button
+            [ class "navbar-toggle"
```
Comment (amitaibu) [inline src/elm/App.elm]: wrong indentation.

File: src/elm/App.elm (hunk 4 above)
Comment (YaronMiro) [inline src/elm/App.elm]: Not yet It will be done on a follow up issue when I tackle the main design.

**Comment by YaronMiro** [comment]
@amitaibu PR is ready.

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**